packages/client/src/deploy.ts

```diff
@@ -66,7 +66,12 @@ async function* postDeployment(
       }
     );
 
-    const deployment = await response.json();
+    let deployment = undefined;
+    try {
+      deployment = await response.json();
+    } catch (error) {
+      throw new Error('Invalid JSON response');
```

Comment:
Can we either (1) debug log the `error` or (2) add the `error` message to the generated error?